### PR TITLE
Feat/dlvr 20/read me

### PR DESCRIPTION
**Pull request by AminaNoMoss**
Populating the code readme. The OR also includes form Cypress config.
Refer to https://nomoss.atlassian.net/browse/DLVR-20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Ionic Tabs Template - Upgraded
+
+This project provides an extension to the tabs template that is provided by Ionic. It provides the following additional functionalities.
+
+- Basic desktop compatible UI
+- Cypress testing
+- WCAG 2.2 AA compliance
```
Comment (latiosu) [inline README.md]: Is this part true? I.e. full WCAG 2.2 AA compliance, that it's been assessed for manual-only testable issues too?

According to a few sources, WCAG 2.2 AA compliance can only partially tested with fully-automatic tools (~20%) and a significant proportion more with semi-automated tools (~60%), the remainder is manual testing (~20%):
- https://karlgroves.com/web-accessibility-testing-what-can-be-tested-and-how/ (2012)
- https://webaim.org/projects/million/ (2024)
- https://www.deque.com/automated-accessibility-testing-coverage/ (2024)

**Review by latiosu — APPROVED** [review]
Awesome work Amina!

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```
Comment (leoankita) [inline cypress/support/commands.ts:1]: What does this file do or intended to do. Its commented, so a bit unclear.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+The following command may be used to run the component tests
+
+```bash
+npm run test:component
+```
+
+To run Cypress component testing on the Cypress GUI, run this command
+
+```bash
+npm run test:component:ui
```
Comment (amnomoss) [inline README.md:92]: We can also add code coverage here. Something like `https://www.npmjs.com/package/@cypress/code-coverage`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+This command will exit with an error if the project has formatting errors. To fix formatting errors, run the following command
+
+```bash
+npm run format-fix
+```
+
+### Running Cypress End to End Tests
+
```
Comment (leoankita) [inline README.md:68]: For the commands like these, I would recommend to use Makefile. It comes in pretty handy.

Comment (leoankita) [reply README.md:68]: Can reference it here -> https://medium.com/finn-no/makefiles-for-frontend-1779be46461b

File: README.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+The following command may be used to run the component tests
+
+```bash
+npm run test:component
+```
+
+To run Cypress component testing on the Cypress GUI, run this command
+
+```bash
```
Comment (leoankita) [inline README.md:93]: We can also add code coverage here. Something like `https://www.npmjs.com/package/@cypress/code-coverage`

**Review by leoankita — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sana-nomoss) [inline README.md:94]: This command is missing from package.json

File: README.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+- Basic desktop compatible UI
+- Cypress testing
+- WCAG 2.2 AA compliance
+- Project formatting (using [Prettier](https://prettier.io/))
+
+Note: the template retains all provisions afforded by the original template as provided by Ionic.
+
+## How to best use this template
+
+This template provides some additional essential elements to help VueJS, Ionic developers get started on their hybrid projects easier and faster.
```
Comment (AminaNoMoss) [inline README.md:16]: James suggestion: "We built this because Ionic, out of the box is not desktop responsive."

File: README.md (hunk 1 above)
Comment (AminaNoMoss) [inline README.md]: Don't say "compliant" since we haven't gotten it audited

File: cypress/support/commands.ts (hunk 2 above)
Comment (AminaNoMoss) [reply cypress/support/commands.ts:1]: Yeah good pickup. It was a file automatically created upon Cypress component testing configuration. I thought I should leave it there as part of Cypress compatibility.

File: README.md (hunk 3 above)
Comment (AminaNoMoss) [reply README.md:94]: Oh thanks!

**Pull request merged**